cmd/clusterctl/pkg/internal/test/fake_objects.go

```diff
@@ -0,0 +1,789 @@
+/*
```

Comment:
Is this something that we should make available more broadly to the repo? I'm not (necessarily) suggesting that we expose it externally, but it might come in handy for unit/integration testing elsewhere within the repository.

Reply:
@detiber now it is designed for testing clusterctl move, but I see your point.

Might be we can merge it as it is, and then eventually move it to the test framework/to somewhere else when there are more clear requirements from the other potential consumers.

Reply:
Let's open an issue so we don't forget later on

Reply:
Here you go!
https://github.com/kubernetes-sigs/cluster-api/issues/1993